pymc_marketing/mmm/lift_test.py

```diff
@@ -643,10 +643,18 @@ def scale_lift_measurements(
         target_transform,
     )
 
-    return pd.concat(
-        [df_lift_test_channel_scaled, df_target_scaled, df_sigma_scaled],
-        axis=1,
-    )
+    if "date" in df_lift_test.columns:  ## for time_varying
+        date_data = df_lift_test[["date"]]
+        components = [
+            df_lift_test_channel_scaled,
+            df_target_scaled,
+            df_sigma_scaled,
+            date_data,
+        ]
+    else:
+        components = [df_lift_test_channel_scaled, df_target_scaled, df_sigma_scaled]
+
+    return pd.concat(components, axis=1)
```

Comment:
It would be great if you could add a test. Maybe @wd60622 can help you with this (I do not know how easy is to add it into the current test-mocking flow :) )